onnx/defs/tensor/defs.cc

```diff
@@ -697,7 +702,11 @@ ONNX_OPERATOR_SET_SCHEMA(
             ctx.getOutputType(0)
                 ->mutable_tensor_type()
                 ->mutable_shape()
-                ->add_dim();
+                ->add_dim()
+                ->set_dim_value(
+                (i < axis) ? data_shape.dim(i).dim_value() : // nb: axis < r
```

Comment:
Is it possible it doesn't have a dim_value?

Reply:
I don't think so. See my comments in 2ff5e38342eca39fc1334d8e242fac0cf681431a